macvtap.go

```diff
@@ -197,16 +224,23 @@ func cmdAdd(args *skel.CmdArgs) error {
 		return err
 	}
 
-	if n.MAC != "" {
+	envArgs, err := getEnvArgs(args.Args)
+	var mac net.HardwareAddr
+	if envArgs != nil {
+		mac, _ = net.ParseMAC(string(envArgs.MAC))
```

Comment:
I think we should check the error here (if that is what you are omitting with _)

Reply:
Right you are. Coming right up.